### DISCUSSION TITLE
[ FAIL ] Starting mcserver: Unable to start LinuxGSM

### DISCUSSION THREAD

**Discussion by adrianschw** [GameServerManagers/LinuxGSM#3671] [Q&A] [answered]
Hey,

I have this error when i will start my minecraft Server

**Comment by johnoclockdk** [comment, marked as answer]
update java to version 16

**Reply by adrianschw** [reply]
Hi there,
First of all thanks for the help, how exactly can I update this?

**Reply by johnoclockdk** [reply]
https://ubuntuhandbook.org/index.php/2021/03/oracle-java-16-released-install-ubuntu-20-04/
